terraform/constellation-cluster/variables.tf

```diff
@@ -1,9 +1,15 @@
+variable "constellation_version" {
+  type        = string
+  description = "Constellation CLI version to use."
```

Comment:
I'm not a big fan of exposing this to the user. The user should not be able to change this default since we don't support it?